examples/seismic/tutorials/07_DRP_schemes.ipynb

```diff
@@ -415,6 +415,7 @@
     }
    ],
    "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
```

Comment:
why are these failing now all of a sudden?

Reply:
Matplotlib changed the default plot size iirc

Reply:
New version of matplotlib yes